x/staking/types/codec.go

```diff
@@ -17,6 +18,11 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgDelegate{}, "cosmos-sdk/MsgDelegate", nil)
 	cdc.RegisterConcrete(&MsgUndelegate{}, "cosmos-sdk/MsgUndelegate", nil)
 	cdc.RegisterConcrete(&MsgBeginRedelegate{}, "cosmos-sdk/MsgBeginRedelegate", nil)
+	cdc.RegisterConcrete(&MsgTokenizeShares{}, "cosmos-sdk/MsgTokenizeShares", nil)
+	cdc.RegisterConcrete(&MsgRedeemTokensForShares{}, "cosmos-sdk/MsgRedeemTokensForShares", nil)
+	cdc.RegisterConcrete(&MsgTransferTokenizeShareRecord{}, "cosmos-sdk/MsgTransferTokenizeShareRecord", nil)
```

Comment:
See my other comment https://github.com/cosmos/cosmos-sdk/pull/16747/files#r1264716595

We are using this naming convention that passed under the limit:
```go
	legacy.RegisterAminoMsg(cdc, &MsgValidatorBond{}, "lsm/MsgValidatorBond")
	legacy.RegisterAminoMsg(cdc, &MsgUnbondValidator{}, "lsm/MsgUnbondValidator")
	legacy.RegisterAminoMsg(cdc, &MsgTokenizeShares{}, "lsm/MsgTokenizeShares")
	legacy.RegisterAminoMsg(cdc, &MsgRedeemTokensForShares{}, "lsm/MsgRedeemTokensForShares")
	legacy.RegisterAminoMsg(cdc, &MsgTransferTokenizeShareRecord{}, "lsm/MsgTransferTokenizeShareRecord")
	legacy.RegisterAminoMsg(cdc, &MsgDisableTokenizeShares{}, "lsm/MsgDisableTokenizeShares")
	legacy.RegisterAminoMsg(cdc, &MsgEnableTokenizeShares{}, "lsm/MsgEnableTokenizeShares")
```

Reply:
looks like only the Transfer function would exceed the limit - should we keep the cosmos-sdk prefix and just change that one to `MsgTransferTokenizeShare`?

Reply:
I'm ok with keepign the prefix, but it just reduces amount of bytes available to describe the stuff. `MsgTransferTokenizeShare` is actually for a record, which is a separate asset and not to be confused with fungible "shares". You meant `MsgTransferTokenizeRecord` ? Sounds good.

Reply:
agreed `MsgTransferTokenizeRecord` is better!

Reply:
updated [[here](https://github.com/cosmos/cosmos-sdk/pull/16747/commits/c65247c6a76d9642d09ddc2761f569fbf4dd8cd3)]